crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -980,6 +978,7 @@ fn check_name(checker: &Checker, expr: &Expr, range: TextRange) {
         _ => None,
     } {
         if is_guarded_by_try_except(expr, module, name, semantic) {
```

Comment:
Same here. It would be great if we could move this check before creating the diagnostic.